provision/group_vars/all

```diff
@@ -0,0 +1,3 @@
+project_name: innovationfund
+document_root: /vagrant
+project_path: "{{ document_root }}/app"
```

Comment:
We usually set project_path to the project root in our CI environment, not a subfolder.